docs/src/content/docs/ref/core-advanced.mdx

```diff
@@ -29,6 +171,110 @@ Usage example:
 let fwdFee: Int = context().readForwardFee();
 ```
 
+:::note[Useful links:]
+
+  [`getOriginalFwdFee(){:tact}`](#getoriginalfwdfee)
+
+:::
+
+## getForwardFee
+
+<Badge text="Available since Tact 1.5" variant="tip" size="medium"/><p/>
+
+```tact
+fun getForwardFee(cells: Int, bits: Int, isMasterchain: Bool): Int;
+```
+
+Calculates and returns the [forward fee][forward-fee] in [nanoToncoins](/book/integers#nanotoncoin) [`Int{:tact}`][int] for an outgoing message consisting of a given number of `cells` and `bits`. Uses the prices of the [masterchain][masterchain] if `isMasterchain` is `true{:tact}`, otherwise the prices of the [basechain][basechain]. The current prices are obtained from the [config param 24 for the masterchain and config param 25 for the basechain][param-24-25] of TON Blockchain.
+
+If both the source and the destination addresses are in the [basechain][basechain], then specify `isMasterchain` as `false{:tact}`. Otherwise, specify `true{:tact}`.
+
+Note, that the values of `cells` and `bits` are taken modulo their maximum values plus $1$. That is, specifying values higher than those listed in [account state limits (`max_msg_cells` and `max_msg_bits`)](/book/exit-codes#50) will have the same result as with specifying the exact limits. In addition, make sure you take [deduplication of cells with the same hash][deduplication] into account.
+
+However, regardless of the values of `cells` and `bits`, this function always adds the minimum price based on the value of [`lump_price`][param-24-25]. See the example for [`getSimpleForwardFee(){:tact}`](#getsimpleforwardfee) to derive it.
```

Comment:
Also one should note that root cell and its bits are not counted in fwd_fee. Fees for that cell are covered by lump_price